.gitignore

```diff
@@ -117,9 +117,11 @@ bin/wp-cli.phar
 /junit.xml
 
 # Cocoapods
-
 ios/Pods
 ios/vendor
 
 # i18n update
-i18n-test/
\ No newline at end of file
+i18n-test/
```

Comment:
[not a blocker] 👋 @fluiddot I'm curious, what is this line for?

Reply:
It's just a test folder that is created when running the command `npm run i18n:update:test`, it's purely for testing purposes.